src/peer_digest.cc

```diff
@@ -382,7 +373,7 @@ peerDigestHandleReply(void *data, StoreIOBuffer receivedData)
         switch (fetch->state) {
 
         case DIGEST_READ_REPLY:
-            retsize = peerDigestFetchReply(fetch, fetch->buf, fetch->bufofs);
+            retsize = peerDigestFetchReply(fetch, fetch->bufofs);
```

Comment:
We removed `fetch->buf` parameters from peerDigestFetchedEnough() and peerDigestFetchReply() calls above, but not from peerDigestSwapInCBlock() and peerDigestSwapInMask() calls below. That inconsistency makes me wonder if the performed removal was out of scope or incomplete. Am I missing something?

If we can significantly reduce the `diff` by (safely) keeping those parameters, then perhaps we should avoid removing those parameters in this PR, at least in peerDigestFetchReply() case.

Reply:
I removed fetch->buf from some functions (ending up with peerDigestReqFinish() call) where it was obviously unused. I undid these changes to minimize PR footprint.